content/applications/finance/fiscal_localizations/philippines.rst

```diff
@@ -61,9 +60,8 @@ For individuals not belonging to a company, identify them by using the following
 BIR 2307 report
 ===============
 
-**BIR 2307** report data, also known as `Certificate of Creditable Tax Withheld at Source
-<https://www.bir.gov.ph/index.php/bir-forms/certificates.html>`_, can be generated for purchase
-orders and vendor payments with the applicable withholding taxes.
+**BIR 2307** report data, also known as *Certificate of Creditable Tax Withheld at Source*,
```

Comment:
Hi @dade-odoo Yes I removed it since the old link was broken. I was thinking to avoid future broken links by reducing number of externally linked documents and also we are currently working on generating the PDF directly in Odoo instead of user to manually fill it in. I think just future proofing a bit as the link may not be needed in the future!

Reply:
I like the idea of removing the link when it's no longer needed, but it seems like at this point, it is still necessary. Once we have the ability to generate the PDF directly in Odoo we'll probably be updating this page again anyway, so that would be a good time to remove the link if you're worried about it breaking again in the future. For now, can you replace it with a working link? Thanks!

Reply:
Thanks! I have added the new link back into the file! I also squashed and commit to this PR hopefully it was successful! Let me know if there is further changes needed!